PyPDF2/__init__.py

```diff
@@ -7,6 +7,7 @@
 You can read the full docs at https://pypdf2.readthedocs.io/.
 """
 
+from ._encryption import PasswordType
```

Comment:
Where / in which scenarios would a PyPDF2 user make use of this?

Reply:
Never mind, I've just seen that it is the return value of a public function :-) :+1: